_data/locales/en.yml

```diff
@@ -131,11 +131,26 @@ technologies:
       collab plugin, which makes it much easier for new users to get 
       started in packaging. It also allows you to work on a single 
       package with the same name from multiple projects, where they 
-      will be updated simultanously.     
+      will be updated simultanously. 
    software:
       trollolo:
         name: trollolo
         url: https://github.com/openSUSE/trollolo
       osc-plugin-collab:
         name: osc-plugin-collab
         url: https://github.com/openSUSE/osc-plugin-collab
+about_title: About openSUSE
+about_description: |
+  openSUSE is a Free Software project created around the openSUSE Linux 
+  distributions and a variety of tools including YaST, the Open Build 
+  Service, openQA,Snapper, KIWI, OSEM and many others. We strive to 
+  promote an open and friendly community where everyone is welcome, 
+  regardless of where they live or work in the world. 
+  
+  openSUSE is well known for it's Linux distributions. Tumbleweed 
+  is a tested, rolling release distribution allowing users to have 
+  an up-to-date system that is tested and well supported. For users 
+  who need long-term support openSUSE maintains Leap, with releases 
```

Comment:
Leap is released every 5ish years with a major release, it is updated with a point version every year, it's a bit confusing I guess